### PR TITLE
Enable Whoops for most integration tests

### PR DESCRIPTION
**Pull request by EreMaijala**
- Also fixes an issue uncovered by this in status-full.phtml.
- Disables Whoops and logging for tests that expect failures.

### CODE REVIEW

File: module/VuFind/tests/integration-tests/src/VuFindTest/Mink/SavedSearchesTest.php [hunk 1]
```diff
@@ -240,7 +240,8 @@ public function testSavedSearchSecurity(): void
         $this->findAndAssertLink($page, 'Log Out')->click();
 
         // Use user A's delete link, but try to execute it as user B:
-        [$base, $params] = explode('?', $delete);
+        [, $params] = explode('?', $delete);
+        $session->setWhoopsDisabled(true);
```
Comment (demiankatz) [inline module/VuFind/tests/integration-tests/src/VuFindTest/Mink/SavedSearchesTest.php:245]:
```suggestion
        $session->setWhoopsDisabled(true); // We expect an error, so let's act like production mode for realistic testing
```

File: module/VuFind/tests/integration-tests/src/VuFindTest/Mink/SavedSearchesTest.php [hunk 2]
```diff
@@ -253,6 +254,7 @@ public function testSavedSearchSecurity(): void
         $this->findAndAssertLink($page, 'Log Out')->click();
 
         // Go back in as user A -- see if the saved search still exists.
+        $session->setWhoopsDisabled(false);
```
Comment (demiankatz) [inline module/VuFind/tests/integration-tests/src/VuFindTest/Mink/SavedSearchesTest.php:258]:
```suggestion
        $session->setWhoopsDisabled(false); // go back to stricter error handling
```

**Review by demiankatz — CHANGES_REQUESTED** [review]
Thanks, @EreMaijala, this looks good to me!

I just thought it might be helpful to add a couple of comments, and I used that as an excuse to try the suggestion feature I mentioned yesterday. Apparently it's not new (at least four years old), but I only just learned about it. It's available as a button through the comment box when you add a comment to one or more lines of code while reviewing.

**Comment by EreMaijala** [comment]
@demiankatz Thanks, and nice to see the suggestions in action. The lines were a bit too long, though, so I decided to add the comments above the actions.

**Review by demiankatz — APPROVED** [review]
Thanks, @EreMaijala!

**Pull request merged**